terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars

```diff
@@ -33,7 +33,7 @@ landing_page_url = "https://www.gov.uk/guidance/register-an-overseas-entity"
 landing_page_starting_new_url = "/register-an-overseas-entity/starting-new"
 payment_fee = "100"
 update_landing_page_url = "/update-an-overseas-entity/continue-with-saved-filing"
-remove_landing_page_url = "TO BE DEFINED"
+remove_landing_page_url = "https://www.gov.uk/guidance/remove-an-overseas-entity"
```

Comment:
Looks fine. The URL gives a 'Page not found' message at the moment. Assume it won't be LIVE until Wednesday.

Reply:
That is the assumpton.

Reply:
Maybe not necessary, but would we need to change the cidev and staging value for remove_landing_page_url?

Reply:
I think that they are probably fine. We don't have a static GOV UK page for those envs, hence why they link somewhere different.